src/test/miner_tests.cpp

```diff
@@ -124,7 +124,10 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
         mempool.addUnchecked(hash, entry.Fee(1000000).Time(GetTime()).SpendsCoinbase(spendsCoinbase).FromTx(tx));
         tx.vin[0].prevout.hash = hash;
     }
-    BOOST_CHECK_THROW(CreateNewBlock(chainparams, scriptPubKey), std::runtime_error);
+    BOOST_CHECK(pblocktemplate = CreateNewBlock(chainparams, scriptPubKey));
```

Comment:
Might be nice to add a comment here since we changed the test behavior from failing to passing with a new defined behavior on the mempool.